perma_web/warc_server/pywb_config.py

```diff
@@ -74,14 +76,24 @@ def get_archive_path():
     # 'unicode' object has no attribute 'get'
     return archive_path.encode('ascii', 'ignore')
 
-def raise_not_found(url):
+
+def raise_not_found(url, timestamp=None):
     if not settings.LOG_PLAYBACK_404:
         # use a custom error to skip pywb printing of exceptions
+        if not timestamp:
+            # if timestamp is not available, fall back on today's date
+            now = datetime.now()
+            # month and day need to be two digits
+            month = str(now.month) if now.month > 9 else "0%s" % now.month
+            day = str(now.day) if now.day > 9 else "0%s" % now.day
+            timestamp = "%s%s%s" % (now.year, month, day)
```

Comment:
Nit: I think you can actually do this all in one bang using the string .format() method, which lets you do fancy stuff like:
```
>>> import datetime
>>> d = datetime.datetime(2010, 7, 4, 12, 15, 58)
>>> '{:%Y-%m-%d %H:%M:%S}'.format(d)
'2010-07-04 12:15:58'
```
Worth it to try  something like that?

Reply:
Ha! Yes! Awesome. I hate that code.